sdks/python/src/redactive/search_client.py

```diff
@@ -33,7 +33,7 @@ async def query_chunks(
         access_token: str,
         semantic_query: str,
         count: int = 1,
-        query_filter: dict | None = None,
```

Comment:
This may affect how the user was using it. We may need to notify  Python SDK users.

Reply:
Agreed, Rob has suggested a deprecation warning (see changes), but for next minor release I'd like to drop `query_filter` for `filters`